apps/meteor/app/utils/lib/mimeTypes.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { expect } from 'chai';
+
+import { getExtension, getMimeType } from './mimeTypes';
+
+const mimeTypeToExtension = {
+	'text/plain': 'txt',
+	'image/x-icon': 'ico',
+	'image/vnd.microsoft.icon': 'ico',
+	'image/png': 'png',
+	'image/jpeg': 'jpeg',
+	'image/gif': 'gif',
+	'image/webp': 'webp',
+	'image/svg+xml': 'svg',
+	'image/bmp': 'bmp',
+	'image/tiff': 'tif',
+	'audio/wav': 'wav',
+	'audio/wave': 'wav',
+	'audio/aac': 'aac',
+	'audio/x-aac': 'aac',
+	'audio/mp4': 'm4a',
+	'audio/mpeg': 'mpga',
+	'audio/ogg': 'oga',
+	'application/octet-stream': 'bin',
+};
+
+const extensionToMimeType = {
+	lst: 'text/plain',
+	txt: 'text/plain',
+	ico: 'image/x-icon',
+	png: 'image/png',
+	jpeg: 'image/jpeg',
+	gif: 'image/gif',
+	webp: 'image/webp',
+	svg: 'image/svg+xml',
+	bmp: 'image/bmp',
+	tiff: 'image/tiff',
+	tif: 'image/tiff',
+	wav: 'audio/wav',
+	aac: 'audio/aac',
+	mp3: 'audio/mpeg',
+	ogg: 'audio/ogg',
+	oga: 'audio/ogg',
+	m4a: 'audio/mp4',
+	mpga: 'audio/mpeg',
+	mp4: 'video/mp4',
+	bin: 'application/octet-stream',
+};
+
+describe('mimeTypes', () => {
+	describe('getExtension', () => {
+		for (const [mimeType, extension] of Object.entries(mimeTypeToExtension)) {
+			it(`should return the correct extension ${extension} for the given mimeType ${mimeType}`, async () => {
+				expect(getExtension(mimeType)).to.be.eql(extension);
+			});
+		}
+
+		it('should return an empty string if the mimeType is not found', async () => {
+			expect(getExtension('application/unknown')).to.be.eql('');
+		});
+	});
+
+	describe('getMimeType', () => {
+		for (const [extension, mimeType] of Object.entries(extensionToMimeType)) {
+			it(`should return the correct mimeType ${mimeType} for the given fileName file.${extension} passing the correct mimeType`, async () => {
+				expect(getMimeType(mimeType, `file.${extension}`)).to.be.eql(mimeType);
+			});
+		}
+
+		it('should return the correct mimeType for the given fileName', async () => {
+			for (const [extension, mimeType] of Object.entries(extensionToMimeType)) {
+				expect(getMimeType('application/unknown', `file.${extension}`)).to.be.eql(mimeType);
```

Comment:
```suggestion
				expect(getMimeType('application/octet-stream', `file.${extension}`)).to.be.eql(mimeType);
```
I believe this incorrect line is what made these tests succeed. `mime.extension('application/unknown')` evaluates to `undefined`,  so we won't get [here](https://github.com/RocketChat/Rocket.Chat/pull/32763/files#diff-e64159623d12c88eb4700ba6e62fc7a89517ee0603d165af21938f9fd899e582R22). However, I believe you meant to have `application/octet-stream` here. In this case, `mime.extension('application/octet-stream')`  returns `bin`

Reply:
No, it's right. If the mime is not in our database it should use the one from the extension

Reply:
Just checked that we now have tests for when the mime is in our database, but it is default one. Looks good to me

Reply:
Just checked that we now have tests for when the mime is in our database, but it is default one. Looks good to me